cmd/janus/init.go

```diff
@@ -18,6 +21,7 @@ var (
 	globalConfig *config.Specification
 	statsClient  stats.Client
```

Comment:
The variables below shouldn't be globally, otherwise we'll face concurrency issues